pkg/cidata/cidata.TEMPLATE.d/lima.env

```diff
@@ -4,6 +4,10 @@ LIMA_CIDATA_MOUNTS={{ len .Mounts }}
 {{- range $i, $val := .Mounts}}
 LIMA_CIDATA_MOUNTS_{{$i}}_MOUNTPOINT={{$val}}
 {{- end}}
+LIMA_CIDATA_DNS={{ len .DNSAddresses }}
+{{- range $i, $val := .DNSAddresses }}
+LIMA_CIDATA_DNS_{{$i}}_ADDRESS={{$val}}
+{{- end}}
```

Comment:
Please add these vars to docs/internal. md

Reply:
The variables are no longer needed; the settings are added to `network-data` for cloud-init instead. That means the Alpine logic will have to be implemented in `lima-init`.